src/auth/auth.service.ts

```diff
@@ -173,7 +175,10 @@ export class AuthService {
 
   toAuthTokenDto(authToken: AuthToken): AuthTokenDto | null {
     if (!authToken) {
-      this.logger.warn(`Recieved ${authToken} argument!`, 'toAuthTokenDto');
+      this.logger.warn(
+        `Recieved ${String(authToken)} argument!`,
```

Comment:
Shouldn't this rather be `authToken.toString()`?

Reply:
That does not work, as `null` and `undefined` do not have `.toString()`